packages/eslint-plugin/src/rules/no-unnecessary-template-expression.ts

```diff
@@ -106,6 +107,10 @@ export default createRule<[], MessageId>({
         if (hasSingleStringVariable) {
           context.report({
             node: node.expressions[0],
```

Comment:
(optional) note that `node` and `loc` are redundant (with `loc` taking precedence), so we can leave `node` off in both these reports.

Reply:
ie,
```suggestion
```

Reply:
Cool, wasn't aware of that, thanks :)